.travis.yml

```diff
@@ -1,6 +1,6 @@
 
 before_install:
-- sudo curl -L -o /usr/local/bin/mill https://github.com/lihaoyi/mill/releases/download/0.2.2/0.2.2 && sudo chmod +x /usr/local/bin/mill
+- sudo curl -L -o /usr/local/bin/mill https://github.com/lihaoyi/mill/releases/download/0.2.3/0.2.3 && sudo chmod +x /usr/local/bin/mill
```

Comment:
@AlexITC Just wanted to point out that the travis script was actually running on mill `0.2.2`. Anyways, bumped version as requested.

Reply:
in fact it was running on `0.2.0`, it uses `0.2.2` in the `develop` branch.